### DISCUSSION TITLE
Delegations list

### DISCUSSION THREAD

**Discussion by MrMoyaert** [adpro99/365ProToolkit#6] [Ideas]
It would be handy to be able to generate a list with all the delegations rights on mailboxes

Example:
**Mailbox                 |    Send as rights    | read and full access**
Shared Mailbox A  | member 1            | Member 1
Shared Mailbox A  | member 2            | Member 2
Shared Mailbox A  | member 3            | Member 4
Shared Mailbox A  | member 4            | 
Shared Mailbox B  | member 1            | Member 1
Shared Mailbox B  | member 2            | Member 2
User Mailbox 1     | member 2            | Member 2

We sometimes get the question "make a new user with the same rights as this user"
So it would be handy to really quick see all the delegations a certain user has

Thanks

**Comment by adpro99** [comment]
Great idea. I will add this to the list. Thanks
